ui/selectors/multichain.ts

```diff
@@ -108,15 +122,18 @@ export function getMultichainNetwork(
 // a popup (for ethereum related stuffs) is being shown (and uses this function), then the native
 // currency will be BTC..
 
-export function getMultichainIsEvm(state: MultichainState) {
+export function getMultichainIsEvm(
+  state: MultichainState,
+  account?: InternalAccount,
+) {
   const isOnboarded = getCompletedOnboarding(state);
   // Selected account is not available during onboarding (this is used in
   // the AppHeader)
-  const selectedAccount = getMaybeSelectedInternalAccount(state);
+  const selectedAccount = account ?? getMaybeSelectedInternalAccount(state);
 
   // There are no selected account during onboarding. we default to the original EVM behavior.
   return (
-    !isOnboarded || !selectedAccount || isEvmAccountType(selectedAccount.type)
+    !isOnboarded ?? !selectedAccount ?? isEvmAccountType(selectedAccount.type)
```

Comment:
This is not correct, we want "logical OR" here

Reply:
Good catch